codechain_primitives/address/AssetAddress.py

```diff
@@ -0,0 +1,13 @@
+from ../Hexstring import H160
+from typing import Tuple, Union
+
+
+class AssetAddress():
+    def __init__(self, addressType: int, payload: Union[H160, str, Tuple], address: str):
+        self.addressType = addressType
+        if type(payload) is H160 or type(payload) is str:
+            self.payload = H160(payload)
+        else:
+            m, n, pubkeys = payload
+            self.payload = (m, n, list(map(lambda p: H160(p), pubkeys)))

```

Comment:
Would you define `MultiSig` type separately and use it rather than just a tuple?
`MultiSig = namedtuple("MultiSig", ["m", "n", "pubkeys"])` would be enough.
3.5: 
```python
MultiSig = typing.NamedTuple("MultiSig", [("m", int), ("n", int), ("pubkeys", List[H160])])
```
3.6: 
```python
class MultiSig(typing.NamedTuple):
	m: int
	n: int
	pubkeys: List[H160]
	def __init__(...)
```
3.7:
```python
@dataclass.dataclass
class MultiSig:
	m: int
	n: int
	pubkeys: List[H160]
	def __init__(...)
```